models/user.server.model.js

```diff
@@ -98,4 +100,4 @@ updateUserById = function(id, updates, done) {
         });
 }
 
-module.exports = { updateUserById, searchUserById, createUser, deleteUserById };
+module.exports = { hashPassword, updateUserById, searchUserById, createUser, deleteUserById };
```

Comment:
```suggestion
module.exports = { hashSaltedPassword, updateUserById, searchUserById, createUser, deleteUserById };
```
I think hashSaltedPassword is better than hashPassword

Reply:
Alright, I will need to refactor this.